docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx

```diff
@@ -36,6 +36,26 @@ Benchmarking helps you test the plotting speed of your farmer against different
 
 Replace /path/to/your/plot with the actual path to your plot. This will provide you with metrics and insights regarding plotting performance.
 
+At the moment, Subspace Node supports two implementation mechanisms:
+
+- `single`, that opens the file once and uses that for reading from multiple threads.
+- `rayon`, that opens files as many times as there are farming threads and, for each thread, uses a dedicated file handle. **Rayon** is the library we use for multi-threading stuff.
+
+To interpret the results: typically, you assess throughput to determine the required size of the SSD for efficient auditing. Both CPU and disk performance influence this metric, so adding a second disk of the same size may not necessarily double the time; its impact is likely to be less significant. Nonetheless, this approach provides you with a general understanding of performance.
+
+To read more about audit benchmarking, [read this forum post](https://forum.subspace.network/t/audit-benchmark-result-explanation-please/2149).
+
+There is a second command available, which helps you determine how much time your farmer has after auditing to provide proof.
+
+```
+./subspace-farmer benchmark prove /path/to/your/plot
+```
+
+Each farmer has ~4 seconds to audit and prove, so depending on how fast auditing is, the remaining time will be used for proving.
+Proving performance doesn’t depend on the plot size.
+
+To read more about prove benchmarking, [read this forum post](https://forum.subspace.network/t/audit-benchmark-result-explanation-please/2150).
+
```

Comment:
Could we add a line about this log message please? We quite often see community members asking what it means.

> If an environment does not have enough time for the proving step, a message such as this is logged by the farmer: 
> ```
> Proving for solution skipped due to farming time limit slot=6723846 sector_index=46
> ```

Reply:
Excellent idea, added!